netlify.toml

```diff
@@ -0,0 +1,13 @@
+[build]
+    base    = "docs"
+    publish = "docs/public"
+    command = "gutenberg build"
+
+[context.production.environment]
```

Comment:
if you don't want to use different binary versions in production and deploy previews, you can replace both `context.production.environment` and `context.deploy-preview.environment` with this single directive:

```
[build.environment]
  GUTENBERG_VERSION = "0.3.0"
```
that will apply to all contexts. If you need to use a specific version for a context, then you can override this value in the specific context.